src/scanner-cli.ts

```diff
@@ -89,7 +89,11 @@ export async function downloadScannerCli(properties: ScannerProperties): Promise
   // Build paths
   const binExt = normalizePlatformName() === 'windows' ? '.bat' : '';
   const dirName = `sonar-scanner-${version}-${normalizePlatformName()}`;
-  const installDir = path.join(SONAR_DIR, SCANNER_CLI_INSTALL_PATH);
+  const installDir = path.join(
+    properties[ScannerProperty.SonarUserHome],
+    SONAR_CACHE_DIR,
+    SCANNER_CLI_INSTALL_PATH,
+  );
```

Comment:
this might not be correct, do we need the CacheDir?

Reply:
We were not using it before, I didn't add it so that we don't lose the cache after the version bump, (and also it doesn't mix md5 folders with hardcoded names?) but both options are fine IMO

Reply:
yeah makes sense to keep it aligned with current version 👍🏻